plugins/modules/win_net_adapter_feature.ps1

```diff
@@ -0,0 +1,127 @@
+#!powershell
+#
+# (c) 2020, ライトウェルの人 <@jirolin>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+#
+# WANT_JSON
+# POWERSHELL_COMMON
+# temporarily disable strictmode, for this module only
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+$ErrorActionPreference = "Stop"
+
+$Interface_name = [string[]]
+$componentID_name = [string[]]
+$params = Parse-Args $args -supports_check_mode $true
+#$Interface_name = Get-AnsibleParam -obj $params -name "Interface" -type "str"
+$Interface_name = Get-AnsibleParam -obj $params -name "Interface"
+$state = Get-AnsibleParam -obj $params -name state -type "str"
+#$componentID_name = Get-AnsibleParam -obj $params -name "componentID" -type "str"
+$componentID_name = Get-AnsibleParam -obj $params -name "componentID"
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+
+$result = @{
+    changed = $false
+}
+
+If(!$Interface_name) { Fail-Json -message "Absolute Interface is not specified for Interface" }
+If(!$state) { Fail-Json -message "Absolute state is not specified for state" }
+If(!$componentID_name) { Fail-Json -message "Absolute componentID is not specified for componentID" }
+
+If ($state -eq 'enable'){
+    $state = $true
+}ElseIf ($state -eq 'disable') {
+    $state = $false
+}Else {
+    Fail-Json -message "Specify the state as 'enable' or 'disable'"
+}
+
+If($Interface_name -is [string]) {
+    If($Interface_name.Length -gt 0) {
+        $Interfaces = @($Interface_name)
+    }
+    Else {
+        $Interfaces = @()
+    }
+}
+Else {
+        $Interfaces = $Interface_name
+}
+
+If($componentID_name -is [string]) {
+    If($componentID_name.Length -gt 0) {
+        $componentIDs = @($componentID_name)
+    }
+    Else {
+        $componentIDs = @()
+    }
+}
+Else {
+    $componentIDs = $componentID_name
+}
+
+Try {
```

Comment:
Don't wrap the code in a blank `try/catch`. Any unhandled exceptions are reported back to Ansible and by catching and calling Fail-Json you are effectively loosing some information that might be useful to the end user.

Reply:
Thank you for pointing it out!
I need to fix this and to execute tests again for transition to ansible collection.